lib/hdiutil.js

```diff
@@ -43,20 +44,20 @@ exports.create = function (volname, size, cb) {
 }
 
 exports.attach = function (path, cb) {
+  const mountpoint = '/Volumes/' + crypto.randomUUID()
```

Comment:
We currently support Node.js >=8.5, `crypto.randomUUID` was added in `14.17.0`.

I have some more breaking changes I want to push out before releasing another major, could you use the `random-path@0.1.2` package? (0.1 support CommonJS, 1.0 is ESM only, I'll upgrade to 1.0 in the next major version 👍)

Something like `npm install random-path@0.1.2`, and then:

```js
const randomPath = require('random-path')
```

```suggestion
  const mountpoint = randomPath('/Volumes', '%s')
```

Reply:
btw. `random-path` is already installed as it's a dependency of `fs-temp`, so that's why I think it's a very good fit for this